src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -954,54 +973,56 @@ void FancyZones::MoveSizeEndInternal(HWND window, POINT const& ptScreen, require
 
 void FancyZones::MoveSizeUpdateInternal(HMONITOR monitor, POINT const& ptScreen, require_write_lock writeLock) noexcept
 {
-    if (m_inMoveSize)
+    if (!m_inMoveSize)

```

Comment:
The only change in MoveSizeUpdateInternal is the inverted logic?
In that case it would be better to do these type of changes in their own PRs.

Reply:
sure, I'll create a separate PR